test/e2e/recipe/index.ts

```diff
@@ -121,8 +121,10 @@ export class RecipeRunner {
             /"@sentry\/electron": ".*"/,
             `"@sentry/electron": "file:./../../../../sentry-electron-${SDK_VERSION}.tgz"`,
           )
-          // We replace the @sentry/tracing dependency version to match that of @sentry/electron
-          .replace(/"@sentry\/tracing": ".*"/, `"@sentry/tracing": "${TRACING_VERSION}"`);
+          // We replace the Sentry JavaScript dependency versions to match that of @sentry/electron
+          .replace(/"@sentry\/tracing": ".*"/, `"@sentry/tracing": "${JS_VERSION}"`)
+          .replace(/"@sentry\/react": ".*"/, `"@sentry/react": "${JS_VERSION}"`)
+          .replace(/"@sentry\/vue": ".*"/, `"@sentry/vue": "${JS_VERSION}"`);
```

Comment:
there is also `@sentry/svelte`, `@sentry/ember`, and `@sentry/angular`!

Reply:
Indeed, although we don't have any example tests for those... yet 😁

I'm also aware that the examples don't actually tests the vue/react SDK functionality!